packages/web/src/services/AudiusBackend.ts

```diff
@@ -109,9 +141,23 @@ export const AuthHeaders = Object.freeze({
   Signature: 'Encoded-Data-Signature'
 })
 
+type SnakeToCamel<S extends string> = S extends `${infer T}_${infer U}`
```

Comment:
lol amazing dude, the new version of ts allowed this yeah?

Reply:
Yup, it's pretty neat. Hopefully this all goes away soon though